protocol/src/common/protocol.ts

```diff
@@ -3448,10 +3448,34 @@ export interface DocumentRangeFormattingParams extends WorkDoneProgressParams {
 	options: FormattingOptions;
 }
 
+/**
```

Comment:
We need a @since tag here and a @proposed tag

Reply:
Done.